proposals/function-pointers.md

```diff
@@ -212,6 +212,29 @@ Restrictions of this feature:
   deliberately not specified by the language. This includes whether they are static vs. instance or
   exactly what signature they are emitted with.
 
+
+### Operators on Function Pointer Types
+
+The section in unsafe code on operators is modified as such:
+
+> In an unsafe context, several constructs are available for operating on all _pointer\_type_s that are not _funcptr\_type_s:
+>
+> *  The `*` operator may be used to perform pointer indirection ([Pointer indirection](unsafe-code.md#pointer-indirection)).
+> *  The `->` operator may be used to access a member of a struct through a pointer ([Pointer member access](unsafe-code.md#pointer-member-access)).
+> *  The `[]` operator may be used to index a pointer ([Pointer element access](unsafe-code.md#pointer-element-access)).
+> *  The `&` operator may be used to obtain the address of a variable ([The address-of operator](unsafe-code.md#the-address-of-operator)).
+> *  The `++` and `--` operators may be used to increment and decrement pointers ([Pointer increment and decrement](unsafe-code.md#pointer-increment-and-decrement)).
+> *  The `+` and `-` operators may be used to perform pointer arithmetic ([Pointer arithmetic](unsafe-code.md#pointer-arithmetic)).
+> *  The `==`, `!=`, `<`, `>`, `<=`, and `=>` operators may be used to compare pointers ([Pointer comparison](unsafe-code.md#pointer-comparison)).
+> *  The `stackalloc` operator may be used to allocate memory from the call stack ([Fixed size buffers](unsafe-code.md#fixed-size-buffers)).
+> *  The `fixed` statement may be used to temporarily fix a variable so its address can be obtained ([The fixed statement](unsafe-code.md#the-fixed-statement)).
+> 
+> In an unsafe context, several constructs are available for operating on all _funcptr\_type_s:
+> *  The `&` operator may be used to obtain the address of static methods ([Allow address-of to target methods](function-pointers.md#allow-address-of-to-target-methods))
```

Comment:
This not operating on funcptr type (contrary to what the previous line says).

Should this rather say that ` The `&` operator may be used to obtain the address of a variable` ?

ie. should the following be possible?

```
delegate*<int> p1 = ...;
var x = &p1;
```

Reply:
You can certainly do that, just like you can do that for any local. The operating on is taken from existing examples for pointer types.

Reply:
> You can certainly do that, just like you can do that for any local. 

Good, it is what I have expected.

The formatting of this section makes it sound like that it is not possible. Taking address of a local is explicitly mentioned in the pointers that are not function pointers section above; but it is not mentioned in this section.